### DISCUSSION TITLE
Error The server time zone value 'Hora est�ndar de Argentina' is unrecognized or represents more than one time zone.

### DISCUSSION THREAD

**Discussion by tobiascastaneda2000** [dds-jv/foro#389] [Consultas de persistencia] [answered]
Buenas Tardes!

EL otro día tratando de correr el runner para la base de datos me surgió el siguiente error:

**Caused by: org.hibernate.exception.JDBCConnectionException: Error calling Driver#connect
...
Caused by: java.sql.SQLException: The server time zone value 'Hora est�ndar de Argentina' is unrecognized or represents more than one time zone. You must configure either the server or JDBC driver (via the serverTimezone configuration property) to use a more specifc time zone value if you want to utilize time zone support.**

La solución que encontré por internet fue agregando al final de la ruta del db lo siguiente: ?serverTimezone=UTC

Por lo tanto, dentro del archivo persistense.xml la linea de la url me quedaria asi: 
`<property name="hibernate.connection.url" value="jdbc:mysql://localhost:3306/huellacarbono?serverTimezone=UTC"/>`

Cuando lo mande a correr ya no me aparecio ese error. Quería preguntar si esta bien la solución o existe una solucion mas recomendable que desconozca.

Dejo esta consulta por si puede ser de ayuda para otro compañero/a que este en la misma situacion.

El enlace donde saque la info: https://es.stackoverflow.com/questions/48935/configurar-zona-horaria-jdbc-driver-java

Saludos!!

**Comment by gastonprieto** [comment, marked as answer]
Buenas @tobiascastaneda2000 

La solución es correcta.
